### DISCUSSION TITLE
2.0.0 Release

### DISCUSSION THREAD

**Discussion by IacobIonut01** [IacobIonut01/Gallery#202] [Announcements]
### Now available on [Google Play](https://play.google.com/store/apps/details?id=com.dot.gallery.gplay) as well (2 different package names, update might take a bit to show on)

- Highly improved memory management
- Fixed more bugs
- Added new UI for Timeline dropdown menu
- Added double tap to seek backwards/forwards on videos
- Added a proper setup screen for first time launch
- Allow users to use material navigation UI instead of the new one
- Allow hiding timeline headers when viewing an album

<hr /><em>This discussion was created from the release <a href='https://github.com/IacobIonut01/Gallery/releases/tag/2.0.0-20070'>2.0.0 Release</a>.</em>

**Comment by fallsdevil** [comment]
incredible, I wanted to know if the "no maps" file is about not having the images' metadata?

**Reply by IacobIonut01** [reply]
It's about not showing location preview (maps image) in media info

**Reply by fallsdevil** [reply]
Got it, thanks... I noticed a bug in this version, when I activate the option to group images by month, When some albums are opened, the app closes unexpectedly, but when you deactivate this function, it works normally.

**Comment by Bastigonzales** [comment]
This is amazing, your app is so much better than most of the pre-installed gallery apps on many Android phones. 💗

**Comment by WSTxda** [comment]
Sugoiiiiiiii 😍😍😍😍
